### ISSUE TITLE
Use buttons instead of div's for the slideshow controls

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#165]
### Problem

[organisms/organisms.slideshow](https://a-ui.github.io/core_branding_scss/#organisms-slideshow)

The controls to navigate to the next or previous slide are empty div's and therefore inaccessible with the keyboard or a screenreader.

```html
<div class="o-slideshow__prev"></div>
<div class="o-slideshow__next"></div>
```

### Solution

Use buttons with hidden text.

### Links and articles

[Understanding Success Criterion 2.1.1: Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)

**Comment by Veyfeyken** [comment]
The controls are buttons now but still empty. A screenreader user can't know what these buttons are for.

```html
<button class="a-button o-slideshow__prev"></button>
```

Suggestion:

* Hide the font icon inside the button with aria-hidden="true"
* Add hidden text `<span class="u-screen-reader-only">previous</span>`

```html
<button class="a-button">
 <span class="o-slideshow__prev" aria-hidden="true"></span>
 <span class="u-screen-reader-only">previous</span>
</button>
```
